src/components/DropdownProblem.tsx

```diff
@@ -81,6 +81,10 @@ export const DropdownProblem = ({
   }
 
   const handleFeedback = (userResponse: string, userAttempts: number): void => {
+    if (userResponse === '') {
+      return
+    }
+
```

Comment:
Clarification question for all of the problems: What happens if this check is removed? (I tested it on MC, but nothing obvious broke :man_shrugging: )

Reply:
Feedback will show if the user submits an answer, resets, and then refreshes the page. Without this you hit the second conditional below. I've included a video example of the behavior without the early return.

```js
if (evaluateInput(userResponse, solution)) {
      setFeedback(correctResponse)
    } else if (retryLimit === 0 || userAttempts !== retryLimit) {
      setFeedback(determineFeedback(userResponse, encourageResponse, answerResponses, evaluateInput))
    } else {
      setFeedback(attemptsExhaustedResponse)
    }
```

https://github.com/openstax/k12-apps-raise/assets/37000585/26b0021d-e4b9-4b4b-bc4e-7806d044f3a7

Reply:
Ah -- right, of course. Thanks!